build.sbt

```diff
@@ -712,11 +706,10 @@ lazy val publishableSettings = Def.settings(
 )
 
 lazy val nonPublishableSettings = Seq(
-  skip in publish := true,
+  publish / skip := true,
   mimaPreviousArtifacts := Set.empty,
-  publishArtifact in (Compile, packageDoc) := false,
-  publishArtifact in packageDoc := false,
-  sources in (Compile, doc) := Seq.empty,
+  Compile / packageDoc / publishArtifact := false,
```

Comment:
Is `packageDoc / publishArtifact := false` not needed?

Reply:
Yep. Even before migration there was a warning from sbt-linter that it isn't used